root/edit/details/AddReleaseGroup.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+
+type AddReleaseGroupProps = {
+  ...EditT,
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string,
+    +name: string,
+    +release_group: ReleaseGroupT,
+    +secondary_types: string,
+    +type: ReleaseGroupTypeT | null,
+  },
+};
+
+const AddReleaseGroup = ({edit}: {+edit: AddReleaseGroupProps}) => {
+  const display = edit.display_data;
+  const type = display.type;
+  const secondaryType = display.secondary_types;
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Release Group'))}</th>
+            <td>
+              <DescriptiveLink entity={display.release_group} />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-release-group">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Name'))}</th>
+            <td>{display.name}</td>
+          </tr>
+
+          <tr>
+            <th>{addColonText(l('Artist'))}</th>
+            <td>
+              <ExpandedArtistCredit
+                artistCredit={display.artist_credit}
+              />
+            </td>
+          </tr>
+
+          {display.comment ? (
+            <tr>
+              <th>{addColonText(l('Disambiguation'))}</th>
+              <td>{display.comment}</td>
+            </tr>
+          ) : null}
+
+          {type ? (
+            <tr>
+              <th>{l('Primary Type:')}</th>
+              <td>
+                {lp_attributes(type.name, 'release_group_primary_type')}
+              </td>
+            </tr>
+          ) : null}
+
+          {secondaryType ? (
+            <tr>
+              <th>{l('Secondary Types:')}</th>
+              <td>
+                {lp_attributes(secondaryType, 'release_group_secondary_type')}
```

Comment:
As you yourself said, you didn't need to add translation stuff here :) (it's coming translated from Perl). Otherwise this looks good to me!

Reply:
Ohh I forgot and added these beforehand, I'll revert this :)